shared/chat/inbox/row/small-team/index.js

```diff
@@ -108,16 +133,6 @@ const styles = styleSheetCreate({
   container: {flexShrink: 0, height: RowSizes.smallRowHeight},
   conversationRow: {
     ...globalStyles.flexBoxColumn,
-    backgroundColor: isMobile ? globalColors.fastBlank : globalColors.blue5,
```

Comment:
removing my fastblank will likely really increase layering on android

Reply:
I've added `fastBlank` back in with `collapseStyles`.